spec/controllers/api/v1/packages_controller_spec.rb

```diff
@@ -46,6 +51,37 @@ def test_orders_packages(package, stockit_request, count)
     end
   end
 
+ describe "PUT move_full_quantity" do
+    before { generate_and_set_token(user) }
+
+    before(:all) do
+      Timecop.freeze('2013-04-24'.to_date) { Time.zone.now.to_date }
```

Comment:
Whenver we will call Time.zone.now.to_date then it will return '2013-04-24'.to_date